src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -43,6 +43,7 @@ const UsersFragment = ({
   const [discovery, discoveryDispatch] = React.useContext(Context).discovery;
   const [profile] = React.useContext(Context).profile;
   const navigation = useNavigation();
+  const [client] = React.useContext(Context).client;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `client` variable is being destructured from the `React.useContext(Context)` hook. However, it's not clear if the `client` variable was previously defined in the context. If it wasn't, this could lead to an undefined value for `client`, which might cause issues when it's used later in the code.
```diff
-   const [client] = React.useContext(Context).client;
+   const { client } = React.useContext(Context);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->